spec/support/authentication_helper.rb

```diff
@@ -1,27 +1,27 @@
 module AuthenticationHelper
-  # def sign_in_as(user)
-    # mock_github_user_auth!(user)
+  def sign_in_as(user)
+    mock_github_user_auth!(user)
 
-    # visit root_path
-    # find(:css, '#sign-in').click
+    visit root_path
+    find(:css, '#sign-in').click
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.